src/components/pages/LandingPrograms/CoderSitters.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Layout, Typography } from 'antd';
+import collage from '../../../styles/ProgramsLandingStyles/assets/codersitters.png';
+const { Content } = Layout;
+const { Title, Text } = Typography;
+
+export default function CoderSitters() {
+  return (
+    <Layout>
+      <Content className="pl__na__sitter__container">
+        <div className="pl__na__top__sitter__container">
+          {/* CODERSITTERS TITLE */}
+          <Title className="pl__na__sitter__title">CODERSITTERS</Title>
+          {/* CODERSITTERS INTRO PARAGRAPH */}
+          <Text className="pl__na__intro__text">
+            Parents get their needed
+            <br />
+            distraction-free time. Kids get to
+            <br />
+            learn coding through play.
+          </Text>
+          {/* BABYSITTER + INSTRUCTOR */}
+          <div className="pl__na__top__mid__container">
+            <Title className="pl__na__sitter__title__container">
+              THINK BABY SITTER + CODING INSTRUCTOR.
+            </Title>
+
+            {/* INFO PARAGRAPH */}
+            <Text className="pl__na__sitter__text__container">
+              Our lessons include playing age-appropriate
+              <br />
+              coding games online, in tandem with a wide
+              <br />
+              range of interactive offline activities. These
+              <br />
+              might include: drawing, prototyping, crafting
+              <br />
+              or imaginative brainstorming
+            </Text>
```

Comment:
Using <br> for line breaks is great

Reply:
line breaks are awesome. It is useful very useful on areas, where the division of lines is significant!